File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/presentation/ui/base/activity/BaseCloseActivity.java
package com.lattis.ellipse.presentation.ui.base.activity;

import android.os.Bundle;
import androidx.annotation.Nullable;

/**
 * Created by Velo Labs Android on 17-04-2017.
 */

public abstract class BaseCloseActivity<Presenter extends ActivityPresenter> extends BaseActivity<Presenter> {
    private final int CLOSE_ICON_WHITE = 03;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);

    }

    @Override
    protected void configureViews() {
        super.configureViews();
        setupAppbarCloseIcon(toolbar, CLOSE_ICON_WHITE);
    }

    @Override
    protected void onResume() {
        super.onResume();


    }


}

File: velolabs_repos/lattis_android_v2/app/src/main/java/com/lattis/lattis/presentation/base/activity/BaseCloseActivity.kt
package com.lattis.lattis.presentation.base.activity

import android.os.Bundle
import com.lattis.lattis.presentation.base.BaseView
import com.lattis.lattis.presentation.ui.base.activity.BaseActivity
import kotlinx.android.synthetic.main.view_toolbar.*

/**
 * Created by Velo Labs Android on 17-04-2017.
 */
abstract class BaseCloseActivity<Presenter : ActivityPresenter<V>,V:BaseView> : BaseActivity<Presenter,V>() {
    private val CLOSE_ICON_WHITE = 3
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
    }

    override fun configureViews() {
        super.configureViews()
        setupAppbarCloseIcon(toolbar, CLOSE_ICON_WHITE)
    }

    override fun onResume() {
        super.onResume()
    }
}